packages/alert-banner/test/alert-banner.test.ts

```diff
@@ -0,0 +1,222 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+import { sendKeys } from '@web/test-runner-commands';
+import { SinonStub, spy, stub } from 'sinon';
+
+import '@spectrum-web-components/alert-banner/sp-alert-banner.js';
+import '@spectrum-web-components/button/sp-button.js';
+import { AlertBanner } from '@spectrum-web-components/alert-banner';
+import {
+    escapeEvent,
+    testForLitDevWarnings,
+} from '../../../test/testing-helpers.js';
+
+describe('AlertBanner', () => {
+    testForLitDevWarnings(
+        async () =>
+            await fixture<AlertBanner>(html`
+                <sp-alert-banner>Your trial will expire soon</sp-alert-banner>
+            `)
+    );
+    it('loads default alert-banner accessibly', async () => {
+        const el = await fixture<AlertBanner>(html`
+            <sp-alert-banner open>Your trial will expire soon</sp-alert-banner>
+        `);
+
+        await elementUpdated(el);
+
+        await expect(el).to.be.accessible();
+    });
+    it('loads variant info', async () => {
+        const el = await fixture<AlertBanner>(html`
+            <sp-alert-banner open variant="info">
+                Your trial will expire soon
+            </sp-alert-banner>
+        `);
+
+        await elementUpdated(el);
+
+        const infoIcon = el.shadowRoot.querySelector('sp-icon-info');
+        expect(infoIcon).to.exist;
+
+        await expect(el).to.be.accessible();
```

Comment:
We are not testing for the element to be accessible here so the assertion here is wrong. Instead you can create a test which will return all the valid variants instead of adding tests for each variant!

Reply:
Done, I've updated the tests